packages/cli/test/acceptance/app-run.acceptance.js

```diff
@@ -14,25 +15,36 @@ const build = require('@loopback/build');
 describe('app-generator (SLOW)', function() {
   const generator = path.join(__dirname, '../../generators/app');
   const rootDir = path.join(__dirname, '../../../..');
-  const sandbox = path.join(rootDir, 'sandbox/sandbox-app');
+  const fullPathSandbox = path.join(rootDir, 'sandbox/sandbox-app');
+  const homeDir = process.env.HOME || process.env.userprofile;
```

Comment:
Is there a reason not to use `os.homedir()` here? It will use the appropriate env var for the platform and also fallback to an appropriate user lookup call (yes, it even reads `/etc/passwd` on UNIX).